openedge.properties

```diff
@@ -1,14 +1,20 @@
 category=Languages
 description=Code Analyzer for OpenEdge
 homepageUrl=https://github.com/Riverside-Software/sonar-openedge
-archivedVersions=
-publicVersions=2.14.4
+archivedVersions=2.14.4
+publicVersions=2.19.1
 
 defaults.mavenGroupId=eu.rssw.sonar.openedge
 defaults.mavenArtifactId=sonar-openedge-plugin
 
 2.14.4.description=New expression engine in ABL Parser
-2.14.4.sqVersions=[8.9,LATEST]
+2.14.4.sqVersions=[8.9,9.6]
```

Comment:
The current version is 9.8. Ending this range with 9.6 means users on 9.7 and 9.8 won't be prompted to upgrade. Is that really what you intend?

Reply:
My understanding was that 2.14.4 (the previous version) would be proposed up to 9.6 (where it has been tested). Only the latest version would be proposed on SQ 9.7+. Am I wrong ?

Reply:
This previous compatibility ends with `LATEST`, which means it's being offered to folks on 9.8. So if you now end compatibility at 9.6, that "strands" your users on 9.7 & 9.8